### PR TITLE
Enable viewing of Public IP address extended location

### PR DESCRIPTION
**Pull request by wdehrich**
## Checklist

- [x] I have read the [_Submitting Changes_](../blob/master/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](../blob/master/CONTRIBUTING.md)
- [x] The title of the PR is clear and informative
- [x] The appropriate `ChangeLog.md` file(s) has been updated:
    - For any service, the `ChangeLog.md` file can be found at `src/{{SERVICE}}/{{SERVICE}}/ChangeLog.md`
    - A snippet outlining the change(s) made in the PR should be written under the `## Upcoming Release` header -- no new version header should be added
- [x] The PR does not introduce [breaking changes](../blob/master/documentation/breaking-changes/breaking-changes-definition.md)
- [x] If applicable, the changes made in the PR have proper test coverage
- [x] For public API changes to cmdlets:
    - [x] a cmdlet design review was approved for the changes in [this repository](https://github.com/Azure/azure-powershell-cmdlet-review-pr) (_Microsoft internal only_)
        - {Please put the link here}
    - [x] the markdown help files have been regenerated using the commands listed [here](../blob/master/documentation/development-docs/help-generation.md#updating-all-markdown-files-in-a-module)

### CODE REVIEW

File: src/Network/Network.Test/ScenarioTests/PublicIpAddressTests.ps1 [hunk 1]
```diff
@@ -819,32 +819,36 @@ function Test-PublicIpAddressCRUD-IdleTimeout
...
+    {
+        # Create the resource group
+        New-AzResourceGroup -Name $resourceGroupName -Location $locationName
+
+        # Create publicIpAddres
+        New-AzPublicIpAddress -ResourceGroupName $resourceGroupName -Name $resourceName -Location $locationName -EdgeZone $edgeZone -AllocationMethod Dynamic -DomainNameLabel $domainNameLabel
+
+	    $publicIP = Get-AzPublicIpAddress -Name $resourceName -ResourceGroupName $resourceGroupName
+	    Assert-AreEqual $publicIP.ExtendedLocation.Name $edgeZone
+        Assert-AreEqual $publicIP.ExtendedLocation.Type 'EdgeZone'
```
Comment (BethanyZhou) [inline src/Network/Network.Test/ScenarioTests/PublicIpAddressTests.ps1:843]: Please correct indents.

**Review by BethanyZhou — CHANGES_REQUESTED** [review]

File: src/Network/Network.Test/ScenarioTests/PublicIpAddressTests.ps1 (hunk 1 above)
Comment (wdehrich) [reply src/Network/Network.Test/ScenarioTests/PublicIpAddressTests.ps1]: fixed

**Review by BethanyZhou — CHANGES_REQUESTED** [review]
Hi @wdehrich , 

1. Please add change log for your change
2. Please update the PR title because it looks like it mismatches with what you are doing.
3. Please check all items in our checklist in PR board

**Review by BethanyZhou — APPROVED** [review]
LGTM

**Review by BethanyZhou — APPROVED** [review]

**Pull request merged**